release/src/main/scripts/build_release_candidate.sh

```diff
@@ -372,6 +372,7 @@ if [[ $confirmation = "y" ]]; then
   cd ${BEAM_ROOT_DIR}
   ./gradlew :sdks:python:generateYamlDocs -PisRelease
   GENERATE_YAMLDOC=~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_YAML_DOC}/${BEAM_ROOT_DIR}/sdks/python/build/yaml-ref.html
+  GENERATE_YAMLDOC_EXAMPLES=~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_YAML_DOC}/${BEAM_ROOT_DIR}/sdks/python/build/yaml-examples.html
```

Comment:
Its fine to leave this, but I think we can actually just get rid of this script.

Reply:
https://github.com/apache/beam/pull/34239 if we can actually clean this up.

Reply:
I merged that, now this PR has merge conflicts, once resolved should be good to merge